core/client/db/src/cache/mod.rs

```diff
@@ -335,3 +334,17 @@ impl<Block: BlockT> BlockchainCache<Block> for DbCacheSync<Block> {
 	}
 }
 
+/// Get pruning strategy for given cache.
+fn cache_pruning_strategy<N: From<u32>>(cache: CacheKeyId) -> PruningStrategy<N> {
+	// the cache is mostly used to store data from consensus engines
+	// this kind of data is only required for non-finalized blocks
```

Comment:
i'm not sure if this would hold true forever. for instance, if we put BABE epoch data in the cache we would care about something from well before the most recently finalized block.

Reply:
however, we don't keep that in the cache so I suppose it's a moot point.